src/utils/tagged-template-literal.js

```diff
@@ -141,6 +141,17 @@ const interleave = (quasis, expressions, absolutePath) => {
        */
       substitute = '$dummyValue'
     }
+    // Make sure substituted by same count of lines
+    const intentCount = prevText.split('\n').pop().length
```

Comment:
By intent you mean indent right? That confused me for so long, can we please change that?

Reply:
And on another note, I'm pretty sure you have some indentation edge cases here with examples such as:
```js
styled.div`
  color: ${colorVariable}
`
```

right? Your length here would include all the `color: ` part

Reply:
And of course adding a test for this as you fix it would also be great!

Reply:
Yes, it's a typo. `intent` should be `indent`.